docassemble/Interpreternotice/data/questions/interpreter_notice_to_include.yml

```diff
@@ -53,9 +53,7 @@ code: |
   # Set the allowed courts for this interview
   allowed_courts = interview_metadata["InterpreterNotice__with_variables"]["allowed courts"]
   nav.set_section('review_InterpreterNotice__with_variables')
-  # Below sets the user_role by asking a question.
-  # You can set user_role directly instead to either 'plaintiff' or 'defendant'
-  user_ask_role
+  user_role
```

Comment:
```suggestion
  user_ask_role
```

Reply:
Given current discussion at https://github.com/SuffolkLITLab/docassemble-AssemblyLine/pull/596 I think `user_ask_role` is the correct variable to invoke here--does that still work if we fix `al_form_type`?

Reply:
Possibly, we can invoke both `user_ask_role` explicitly followed by `user_role` to make sure the value is defined

Reply:
I don't think we need to invoke `user_role`, that was in prep for the earlier version of https://github.com/SuffolkLITLab/docassemble-AssemblyLine/pull/596 to be merged. Now that we're avoiding `user_role`, `user_ask_role` should be fine.